spyro/domains/quadrature.py

```diff
@@ -46,68 +44,11 @@ def quadrature_rules(V):
         qr_s = {}
         qr_k = {}
     elif (cell_geometry in {quadrilateral, hexahedron, TensorProductCell(quadrilateral, interval)}
-            and family <= {"Q", "DQ", "TensorProductElement"}):
-        dimension = cell_geometry._tdim
+            and family <= {"Q", "DQ"}):
         # In this case, for the spectral element method we use GLL quadrature
-        qr_x_rule = gauss_lobatto_legendre_cube_rule(
-            dimension=dimension, degree=degree
-        )
-        qr_s_rule = gauss_lobatto_legendre_cube_rule(
-            dimension=(dimension - 1), degree=degree
-        )
-        # Convert to dictionary format for consistent interface
-        qr_x = {"scheme": qr_x_rule}
-        qr_k = {"scheme": qr_x_rule}
-        qr_s = {"scheme": qr_s_rule}
+        qr_x = {"scheme": "lump", "degree": degree}
```

Comment:
We could take advantage of this PR to document what is the range of degrees available for 2D and 3D quadratures.